packages/caip/src/utils.ts

```diff
@@ -51,55 +34,13 @@ export const getFeeAssetIdFromAssetId = (assetId: AssetId): AssetId | undefined
 export const isValidChainPartsPair = (
   chainNamespace: ChainNamespace,
   chainReference: ChainReference
-) => VALID_CHAIN_IDS[chainNamespace]?.includes(chainReference) || false
+) => constants.VALID_CHAIN_IDS[chainNamespace]?.includes(chainReference) || false
 
-export const makeBtcData = () => {
-  const chainNamespace = CHAIN_NAMESPACE.Bitcoin
-  const chainReference = CHAIN_REFERENCE.BitcoinMainnet
-  const assetId = toAssetId({
-    chainNamespace,
-    chainReference,
-    assetNamespace: 'slip44',
-    assetReference: ASSET_REFERENCE.Bitcoin
-  })
-  return { [assetId]: 'bitcoin' }
-}
-
-export const makeDogecoinData = () => {
-  const chainNamespace = CHAIN_NAMESPACE.Bitcoin
-  const chainReference = CHAIN_REFERENCE.DogecoinMainnet
-  const assetId = toAssetId({
-    chainNamespace,
-    chainReference,
-    assetNamespace: 'slip44',
-    assetReference: ASSET_REFERENCE.Dogecoin
-  })
-  return { [assetId]: 'dogecoin' }
-}
-
-export const makeCosmosHubData = () => {
-  const chainNamespace = CHAIN_NAMESPACE.Cosmos
-  const chainReference = CHAIN_REFERENCE.CosmosHubMainnet
-  const assetId = toAssetId({
-    chainNamespace,
-    chainReference,
-    assetNamespace: 'slip44',
-    assetReference: ASSET_REFERENCE.Cosmos
-  })
-  return { [assetId]: 'cosmos' }
-}
-
-export const makeOsmosisData = () => {
-  const chainNamespace = CHAIN_NAMESPACE.Cosmos
-  const chainReference = CHAIN_REFERENCE.OsmosisMainnet
-  const assetId = toAssetId({
-    chainNamespace,
-    chainReference,
-    assetNamespace: 'slip44',
-    assetReference: ASSET_REFERENCE.Osmosis
-  })
-  return { [assetId]: 'osmosis' }
-}
+export const bitcoinAsset = { [constants.btcAssetId]: 'bitcoin' }
```

Comment:
`bitcoinAssetMap` or `bitcoinAssetData` - flip a coin

Reply:
addressed in https://github.com/shapeshift/lib/pull/914/commits/080c21f5f8f5aa8c96b55e31126c9b828da30d52